gatorgrade/main.py

```diff
@@ -16,22 +20,23 @@ def gatorgrade(
     """Run the GatorGrader checks in the gatorgrade.yml file."""
     # check if ctx.subcommand is none
     if ctx.invoked_subcommand is None:
-        if filename.suffix == "yml":
-            pass
+        checks = parse_config(FILE)
+        run_and_display_command_checks(checks)
 
 
 @app.command()
 def generate(
+    target_path_list: List[str],
     force: bool = typer.Option(
         False,
         "--force",
         "-f",
-        help="Force gatorgrade to overwrite an existing yml file.",
-    )
+        help="Force gatorgrade to overwrite an existing yml file."
+    )  
 ):
     """Generate a gatorgrade.yml file."""
     if force is False:
-        pass
+        generate_config(target_path_list)
```

Comment:
If `--force` isn't implemented, you should just remove the code related to it and instead directly call `generate_config` without any `if` statement.